build_container/build_container_ubuntu.sh

```diff
@@ -92,19 +92,19 @@ PACKAGES=(
 apt-get install -y --no-install-recommends "${PACKAGES[@]}"
 
 # Set LLVM version for each cpu architecture.
-LLVM_VERSION=12.0.0
+LLVM_VERSION=12.0.1
 case $ARCH in
     'ppc64le' )
         LLVM_DISTRO=powerpc64le-linux-ubuntu-18.04
-        LLVM_SHA256SUM=acf6bd491d60d126254dc867b2c64c9f1077a4b3dabc0381c31fd8e72aacc4f2
+        LLVM_SHA256SUM=271b9605b74d904d3cc05dd6a61e927fd5a46d5f6b7541cdc67186eb02b22e4c
         ;;
     'x86_64' )
         LLVM_DISTRO=x86_64-linux-gnu-ubuntu-16.04
-        LLVM_SHA256SUM=9694f4df031c614dbe59b8431f94c68631971ad44173eecc1ea1a9e8ee27b2a3
+        LLVM_SHA256SUM=6b3cc55d3ef413be79785c4dc02828ab3bd6b887872b143e3091692fc6acefe7
```

Comment:
With ubuntu 16.04 now expired in April, should this be kicked up to 18.04 LTS?

Reply:
That's what LLVM release provides, and it works on 16.04+. our image base are 18.04.